frontend/eslint.config.ts

```diff
@@ -65,13 +65,13 @@ export default defineConfigWithVueTs(
       'vue/eqeqeq': 'error',
       'vue/match-component-file-name': 'error',
       'vue/match-component-import-name': 'error',
+      'vue/multi-word-component-names': 'off',
```

Comment:
I give up on bothering with this, I get their [point](https://eslint.vuejs.org/rules/multi-word-component-names#rule-details) but the use case will never apply to us and when the component is `Button` or in this case `Stepper` I don't have anything good to add other than `TButton` or `TStepper` 🫠